### ISSUE TITLE
Feature request: config option to disable 2 factor nag screen

### ISSUE THREAD

**Issue by getut** [Ylianst/MeshCentral#3608]
In combination with the new config option 2falock, we would like to disable the backup codes and have users only have a single 2 factor route enabled at any given time. Admins with server access are at no additional risk of being locked out of the server, but we would greatly prefer that users only have a single 2 factor option available without being nagged at each sign on.

**Comment by Ylianst** [comment]
Working on this now.

**Comment by Ylianst** [comment]
Just released with MeshCentral v0.9.79. In the domain section of the config.json, try this:

```
      "PasswordRequirements": {
        "backupcode2factor": false,
        "single2factorWarning": false,
      }
```

Let me know if it works.

**Comment by getut** [comment]
That got it! Thank you!
